### DISCUSSION TITLE
docs/v1.10.x/guide/vpc/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1818] [Announcements]
# docs/v1.10.x/guide/vpc/

https://kubeovn.github.io/docs/v1.10.x/guide/vpc/

**Comment by zhangcongjiang** [comment]
建议官网对于配置样例的正确结果给出一些校验机制，例如配置完成后如何校验配置的正确性，通过kubctl-ko查看到哪些数据，对应的pod上有哪些路由条目，转发过程是什么样子的

**Reply by oilbeater** [reply]
感谢建议，我们会完善这块的文档

**Comment by hongzhen-ma** [comment]
创建 VPC 网关并配置默认路由   中最后一行 “nextHopIP：需和 lanIp 相同”
应该是放在 “自定义路由/静态路由” 下
